upgrade/upgrade-1.3.php

```diff
@@ -0,0 +1,170 @@
+	}
+	
+	if (in_array('ZELERIS_IMPORTE_MINIMO_ENVIO_GRA',$config_installed))
+	{
+		Configuration::updateValue('ZELERIS_MIN_AMOUNT_FREE', Configuration::get('ZELERIS_IMPORTE_MINIMO_ENVIO_GRA'));
+		Configuration::deleteByName('ZELERIS_IMPORTE_MINIMO_ENVIO_GRA');
+	}
+	
+	if (in_array('ZELERIS_IMPORTE_MINIMO_ENVIO_G_I',$config_installed))
+	{
+		Configuration::updateValue('ZELERIS_MIN_AMOUNT_FREE_INT', Configuration::get('ZELERIS_IMPORTE_MINIMO_ENVIO_G_I'));
+		Configuration::deleteByName('ZELERIS_IMPORTE_MINIMO_ENVIO_G_I');
+	}
+	
+	if (in_array('ZELERIS_COSTE_MANIPULACION',$config_installed))
+	{
+		Configuration::updateValue('ZELERIS_MANIPULATION_COST', Configuration::get('ZELERIS_COSTE_MANIPULACION'));
+		Configuration::deleteByName('ZELERIS_COSTE_MANIPULACION');
+	}
+
+	
+	
+	$fields = DB::getInstance()->executeS('DESCRIBE '._DB_PREFIX_.'zeleris_email');
+	if (is_array($fields))
+		foreach($fields as $field)
+			$field_installed[] = $field['Field'];
+			
+	if (!in_array('title',$field_installed))
+	{
+		$sql='ALTER TABLE '._DB_PREFIX_.'zeleris_email CHANGE `titulo` `title` VARCHAR( 128 ) CHARACTER SET utf8 COLLATE utf8_general_ci NULL DEFAULT NULL';
+		Db::getInstance()->execute($sql);
+	}
+	
+	if (!in_array('message',$field_installed))
+	{
+		$sql='ALTER TABLE '._DB_PREFIX_.'zeleris_email CHANGE `mensaje` `message` TEXT CHARACTER SET utf8 COLLATE utf8_general_ci NULL DEFAULT NULL ';
+		Db::getInstance()->execute($sql);
+	}
+	
+	
+	$sql='SHOW TABLES LIKE "'._DB_PREFIX_.'zeleris_envios"';
+	$result = Db::getInstance()->executeS($sql);
+	
+	if($result)
+	{
+		$sql='RENAME TABLE '._DB_PREFIX_.'zeleris_envios TO '._DB_PREFIX_.'zeleris_orders';
+		Db::getInstance()->execute($sql);
+		
+		$fields = DB::getInstance()->executeS('DESCRIBE '._DB_PREFIX_.'zeleris_orders');
+		
+		if (is_array($fields))
+			foreach($fields as $field)
+				$field_installed[] = $field['Field'];
+				
+				
+		if (in_array('id_envio',$field_installed))
+		{
+			$sql='ALTER TABLE '._DB_PREFIX_.'zeleris_orders CHANGE `id_envio` `id_zeleris_orders` INT( 11 ) NOT NULL AUTO_INCREMENT';
+			Db::getInstance()->execute($sql);
+		}
+		
+		if (in_array('id_envio_order',$field_installed))
+		{
+			$sql='ALTER TABLE '._DB_PREFIX_.'zeleris_orders CHANGE `id_envio_order` `id_order` INT( 11 ) NOT NULL';
+			Db::getInstance()->execute($sql);
+		}
+		
+		if (in_array('codigo_envio',$field_installed))
+		{
+			$sql='ALTER TABLE '._DB_PREFIX_.'zeleris_orders CHANGE `codigo_envio` `send_code` VARCHAR( 50 ) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL';
+			Db::getInstance()->execute($sql);
+		}
+		
+		if (in_array('fecha',$field_installed))
+		{
+			$sql='ALTER TABLE '._DB_PREFIX_.'zeleris_orders CHANGE `fecha` `date` DATETIME NOT NULL';
+			Db::getInstance()->execute($sql);
+		}
+		
+		if (!in_array('packages',$field_installed))
+		{
+			$sql='ALTER TABLE '._DB_PREFIX_.'zeleris_orders ADD packages int(11) NOT NULL';
+			Db::getInstance()->execute($sql);
+		}
+	}
+	
+	return $module;
```

Comment:
You must return a boolean. `true` if the upgrade succeed, or `false` if something went wrong.